day_two_guide/topics/deprecating_master.adoc

```diff
@@ -0,0 +1,175 @@
+////
+Deprecating a Master Host
+
+Module included in the following assemblies:
+
+* day_two_guide/host_level_tasks.adoc
+////
+
+Deprecating a master host removes it from the {product-title} environment.
+
+The reasons to deprecate or scale down master hosts include hardware re-sizing
+or replace the underlying infrastructure.
+
+Highly available {product-title} environments require at least three master
+hosts and three etcd nodes. Usually, the master hosts are collocated with the
+etcd services. This topic describes the deprecation process for master hosts
+with or without collocated etcd.
+
+You should create a backup of the configuration and data files prior to any
+important task such as deprecating a master host. See the
+xref:../day_two_guide/host_level_tasks.adoc#day-two-guide-master-backup[Creating
+a master host backup] and
+xref:../day_two_guide/host_level_tasks.adoc#day-two-guide-etcd-backup[etcd
+tasks] sections for more information.
+
+[IMPORTANT]
+====
+Ensure the master services and etcd always be deployed in odd numbers due to the
```

Comment:
I think you mean:

"Ensure that the master services and `etcd` are always deployed in odd numbers..."